### DISCUSSION TITLE
Share apps over local wifi

### DISCUSSION THREAD

**Discussion by trymeouteh** [ImranR98/Obtainium#1761] [Ideas]
An option inside the app to share downloaded and installed apps to other devices on the same Wifi network.

To have a new tab on the app that will allow you to start up a HTTP server. When this HTTP server is launches, you can choose what APK with the JSON files to share and it will display your LAN IP address and display a QR code of your IP address. On the other device, they can use their web browser to go to the LAN IP address and download the apps that are being shared or use the Obtainium app and go to the share tab to connect to a nearby device and download the APKs with the JSON data.

**Comment by ImranR98** [comment]
This doesn't make sense for Obtainium to implement because it's very complex (you'd need to deal with various network issues, VPNs, etc.) and there are dedicated apps like [LocalSend](https://localsend.org/) and [KDE Connect](https://kdeconnect.kde.org/) that do this much better than Obtainium ever could. You can share exports from Obtainium to those apps.

**Reply by trymeouteh** [reply]
Where are the APK stored on the device that are downloaded from the Obtainium app?
